app/components/TableMenu/index.js

```diff
@@ -79,6 +80,13 @@ class TableMenu extends React.Component {
         disabled: false,
       },
       {
+        name: 'mute',
```

Comment:
The pretty strange place for it. Let's move it to `Sit-out/Stand-up` menu.

Reply:
done